Makefile-libs.Inc

```diff
@@ -119,8 +119,8 @@ $(BINDIR_ATHENA_SETUP_LIBS): $(BIN_DIR).athena.version.$(ATHENA_SETUP_REV)
 
 # rename athena VM library
 ATHENA_LIB_RENAMED := libathenavmwrapper$(suffix $(ATHENA_LIB))
-
-$(BIN_DIR)$(ATHENA_LIB_RENAMED): $(BIN_DIR)$(ATHENA_LIB)
+ATHENA_LIB_PATH := $(BIN_DIR)$(ATHENA_LIB_RENAMED)
```

Comment:
That may confuse someone as most of the time env variables with PATH suffix contain list of directories to search for an entry.
IMO better name would be ATHENA_LIB_LOCATION.